Cloud changelog - Aug 31st release (#6589)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2023-08-31](#release-2023-08-31)
- [Release 2023-08-09](#release-2023-08-09)
- [Release 2023-08-03](#release-2023-08-03)
- [Release 2023-07-26](#release-2023-07-26)
- [Release 2023-07-20](#release-2023-07-20)
- [Release 2023-07-19](#release-2023-07-19)

## Release 2023-08-31

### Improvements

#### User Interface (UI)
 - Added functionality to bulk mark a whole channel category as read.
 - Removed Boards product tour code.

#### Administration
 - API examples are now updated to reflect latest Go API conventions, deprecating older code samples.
 - Updated the public server module version to v0.0.7.
 - Added a ``Post Action`` plugin hook to allow plugins to register new items in the post menu.
 - Added a ``Post Editor Action`` plugin hook to allow plugins to register new items in the post editor menu.

>  - Improved logging on plugin initialization, activation and removal.

Reviewer: ```suggestion
 - Improved logging on plugin initialization, activation, and removal.
```

 - Removed the deprecated ``ManifestExecutables`` struct.
 - Removed the deprecated ``UserAuth.Password`` field.

>  - The server is now built using a node version matching server.

Reviewer: ```suggestion
 - The server is now built using a node version matching the server version.
```

Latest Mattermost Cloud releases:

- [Release 2023-08-31](#release-2023-08-31)
- [Release 2023-08-09](#release-2023-08-09)
- [Release 2023-08-03](#release-2023-08-03)
- [Release 2023-07-26](#release-2023-07-26)
- [Release 2023-07-20](#release-2023-07-20)
- [Release 2023-07-19](#release-2023-07-19)

## Release 2023-08-31

### Improvements

#### User Interface (UI)
 - Added functionality to bulk mark a whole channel category as read.
 - Removed Boards product tour code.

#### Administration
 - API examples are now updated to reflect latest Go API conventions, deprecating older code samples.
 - Updated the public server module version to v0.0.7.
 - Added a ``Post Action`` plugin hook to allow plugins to register new items in the post menu.
 - Added a ``Post Editor Action`` plugin hook to allow plugins to register new items in the post editor menu.
 - Improved logging on plugin initialization, activation, and removal.
 - Removed the deprecated ``ManifestExecutables`` struct.
 - Removed the deprecated ``UserAuth.Password`` field.

>  - The server is now built using a node version matching the server version.

Reviewer 2: This sentence doesn't look right.

Author: It came from this PR https://github.com/mattermost/enterprise/pull/1527.
Should it be "The enterprise is now built...."? cc @lieut-data @agnivade

Reviewer 3: A release note was probably premature on my part: propose we just remove this instead of delving into the nuances of the fact that our build tooling was being invoked with a slightly out of sync Node version when building the enterprise images.

Author: Removed 👍

Latest Mattermost Cloud releases:

- [Release 2023-08-31](#release-2023-08-31)
- [Release 2023-08-09](#release-2023-08-09)
- [Release 2023-08-03](#release-2023-08-03)
- [Release 2023-07-26](#release-2023-07-26)
- [Release 2023-07-20](#release-2023-07-20)
- [Release 2023-07-19](#release-2023-07-19)

## Release 2023-08-31

### Improvements

#### User Interface (UI)
 - Added functionality to bulk mark a whole channel category as read.
 - Removed Boards product tour code.
 - Replaced Gfycat with Giphy in the gif picker.
 - Updated Focalboard plugin version to 7.11.3.
 - Pre-packaged Playbooks version 1.38.1.

>  - Upgraded prepackaged Zoom plugin to v1.6.2, and Antivirus plugin to v1.0.0.

Reviewer: @esethna - Is Antivirus going to be a pre-packaged plugin moving forward?

Author: cc @calebroseland

Reviewer 4: @cwarnermm see https://github.com/mattermost/mattermost/pull/24311#issuecomment-1695779562 for a much longer explanation

Author: I've removed the Antivirus plugin note from this PR based on https://github.com/mattermost/docs/pull/6589#pullrequestreview-1598397709

 - Added a ``Post Action`` plugin hook to allow plugins to register new items in the post menu.
 - Added a ``Post Editor Action`` plugin hook to allow plugins to register new items in the post editor menu.
 - Improved logging on plugin initialization, activation, and removal.
 - Removed the deprecated ``ManifestExecutables`` struct.
 - Removed the deprecated ``UserAuth.Password`` field.
 - Remote users are no longer counted as part of the license.
 - Improved data retention logs.
 - Removed ``/opengraph`` endpoint as it was unused.
 - Transitionally prepackaged plugins are now installed to the filestore for continuity when a future release stops prepackaging those plugins.
 - Removed the deprecated ``Manifest.RequiredConfig`` field.
 - Added a ``NotificationWillBePushed`` plugin hook invoked before the push notification is processed and sent to the notification service. Plugins may modify or reject the push notification.
 - Added a `SendPushNotification` plugin api method which allows plugins to send push notifications to a specific user's mobile sessions.
 - Disabled ``PluginSettings.EnableRemoteMarketplace`` functionality.

### Bug Fixes

>  - Fixed accessibility issues: tab support at login, reset and signup pages, and App bar.

Reviewer: ```suggestion
 - Fixed accessibility issues: tab support at login, reset pages, sign up pages, and App bar.
```

 - Fixed an issue where CRLF line endings passed to mmctl commands were not being stripped from commands.

>  - Fixed an issue where text copied from Microsoft OneNote pastes it as an image.

Reviewer: ```suggestion
 - Fixed an issue where text copied from Microsoft OneNote is pasted as an image.
```

Latest Mattermost Cloud releases:

- [Release 2023-08-31](#release-2023-08-31)
- [Release 2023-08-09](#release-2023-08-09)
- [Release 2023-08-03](#release-2023-08-03)
- [Release 2023-07-26](#release-2023-07-26)
- [Release 2023-07-20](#release-2023-07-20)
- [Release 2023-07-19](#release-2023-07-19)

## Release 2023-08-31

### Improvements

#### User Interface (UI)
 - Added functionality to bulk mark a whole channel category as read.
 - Removed Boards product tour code.
 - Replaced Gfycat with Giphy in the gif picker.
 - Updated Focalboard plugin version to 7.11.3.
 - Pre-packaged Playbooks version 1.38.1.

>  - Upgraded prepackaged Zoom plugin to v1.6.2, and Antivirus plugin to v1.0.0.

Reviewer 4: @amyblais I would recommend removing the reference to Antivirus for cloud-specific release notes. #24336's release note was on-prem inclusive.

```suggestion
 - Upgraded prepackaged Zoom plugin to v1.6.2.
```
